build-config/make/images.make

```diff
@@ -272,6 +272,7 @@ $(SYSROOT_COMPLETE_STAMP): $(SYSROOT_CHECK_STAMP)
 	$(Q) echo "onie_build_machine=$(ONIE_BUILD_MACHINE)" >> $(MACHINE_CONF)
 	$(Q) echo "onie_machine_rev=$(MACHINE_REV)" >> $(MACHINE_CONF)
 	$(Q) echo "onie_arch=$(ARCH)" >> $(MACHINE_CONF)
+	$(Q) echo "onie_build_platform=$(ARCH)-$(ONIE_BUILD_MACHINE)-$(MACHINE_REV)" >> $(MACHINE_CONF)
```

Comment:
The letter `r` is lost in the string.  i.e. it would be `$(ARCH)-$(ONIE_BUILD_MACHINE)-r$(MACHINE_REV)`.   😃

Reply:
ah ha!  good catch. i will update that.